sdk/src/trace/batch_span_processor.cc

```diff
@@ -221,6 +224,15 @@ void BatchSpanProcessor::NotifyCompletion(
 
   if (notify_force_flush)
   {
+    if (exporter)
+    {
+      std::chrono::microseconds timeout = opentelemetry::common::DurationUtil::AdjustWaitForTimeout(
+          std::chrono::microseconds{
+              synchronization_data->force_flush_timeout_us.load(std::memory_order_acquire)},
+          std::chrono::microseconds::zero());
+      exporter->ForceFlush(timeout);
```

Comment:
Just to confirm, here we are invoking `ForceFlush` on exporter once the `ForceFlush` on processor is completed?

Reply:
I meant we don't need this method for the exporters which don't have anything to flush, let the base class method to get called by default.